lnbits/core/crud.py

```diff
@@ -16,7 +16,10 @@ def create_account() -> User:
         user_id = uuid4().hex
         db.execute("INSERT INTO accounts (id) VALUES (?)", (user_id,))
 
-    return get_account(user_id=user_id)
+    new_account = get_account(user_id=user_id)
+    assert new_account, "Newly created account couldn't be retrieved"
```

Comment:
There are a few places where I used this pattern, I'm not sure if a function `expect(x: Option[T], assert_msg: str) -> T` would be a good abstraction in the python world (I'd totally go for it as a rust programmer :smile:).